ci-operator/jobs/infra-origin-release-images.yaml

```diff
@@ -0,0 +1,282 @@
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        oc -n ci start-build origin-release-nodejs-8 --wait=true
+      command:
+      - /bin/bash
+      image: registry.ci.openshift.org/ocp/4.7:cli
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 500m
+    serviceAccountName: origin-release-images-builder
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+  max_concurrency: 1
+  name: periodic-build-origin-release-image-origin-release-nodejs-8-browser-tests
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        oc -n ci start-build origin-release-nodejs-8-browser-tests --wait=true
+      command:
+      - /bin/bash
+      image: registry.ci.openshift.org/ocp/4.7:cli
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 500m
+    serviceAccountName: origin-release-images-builder
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+  max_concurrency: 1
+  name: periodic-build-origin-release-image-origin-release-rhel-7-golang-1.11
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        oc -n ci start-build origin-release-rhel-7-golang-1.11 --wait=true
+      command:
+      - /bin/bash
+      image: registry.ci.openshift.org/ocp/4.7:cli
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 500m
+    serviceAccountName: origin-release-images-builder
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+  max_concurrency: 1
+  name: periodic-build-origin-release-image-origin-release-rhel-7-golang-1.12
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        oc -n ci start-build origin-release-rhel-7-golang-1.12 --wait=true
+      command:
+      - /bin/bash
+      image: registry.ci.openshift.org/ocp/4.7:cli
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 500m
+    serviceAccountName: origin-release-images-builder
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+  max_concurrency: 1
+  name: periodic-build-origin-release-image-origin-release-rhel-8-golang-1.12
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
```

Comment:
Do we even need these to be this complicated when they just run `oc`?

Reply:
Updated.